mautic/mautic
# Fixed populating the campaign status on form action push to integration

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

When using the salesforce push to integration form action, the campaign member status was lost on save because the form name was incorrectly parsed. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create/edit a push to integration form action and select salesforce, choose a campaign, then choose a status
2. Save and reopen the action and note the status is lost

#### Steps to test this PR:
1. Apply the PR, refresh the browser cache (open dev console), and repeat. This time status should be restored
